udfs/datasketches/README.md

```diff
@@ -0,0 +1,291 @@
+| Aggregate | **FunctionName**: theta_sketch_intersection(theta_sketch) <br> **Input**: Sketch Bytes <br> **Output**: sketch Bytes<br> **Description**: Aggregates multiple theta sketches, performs an intersection op and returns a merged theta sketch                                                                                                                                                                |
+| Scalar    | **FunctionName**: theta_sketch_a_not_b(theta_sketch_a, theta_sketch_b ) <br> **Input**: Sketch Bytes <br> **Output**: sketch Bytes<br> **Description**: Takes in 2 theta sketches, performs a difference op / a_not_b op (i.e SetA - SetB)  and returns a theta_sketch                                                                                                                                     |
+| Scalar    | **FunctionName**: theta_sketch_extract(theta_sketch) <br> **Input**: theta_sketch -> Bytes <br> **Output**: FLOAT64 <br> **Description**: Takes in a theta sketch, returns approx distinct count of entries of the id_col used to create the theta sketch                                                                                                                                                  |
+
+### Lg_k - Precision parameter
+
+Lg_k is one of the parameters in select theta sketch functions.    
+Choice of lg_k int64 parameter is important as it has direct correlation between your query slot usage, theta_sketch size and relative error.
+- Lg_k vs K vs Relative error comparison can be found in [this public doc](https://datasketches.apache.org/docs/Theta/ThetaErrorTable.html)
+- theta_sketch size vs K comparison can be found in [this public doc](https://datasketches.apache.org/docs/Theta/ThetaSize.html)
+
+### Examples
+
+1. Generates 2 tables with sample data ( treated as SetA, setB)
+
+```sql
+-- Generate ints from [1, 5M]
+CREATE OR REPLACE TABLE `$BQ_PROJECT.$BQ_DATASET`.set_a AS
+SELECT
+    CONCAT("group_key_", CAST(RAND()*10 AS INT64)) as group_key,
+    1000000*x2 + x1 as x
+from UNNEST(GENERATE_ARRAY(1,1000000)) as x1,
+     UNNEST(GENERATE_ARRAY(0,4)) as x2;
+
+-- Generate ints from [4M, 7M]
+CREATE OR REPLACE TABLE  `$BQ_PROJECT.$BQ_DATASET`.set_b AS
+SELECT
+    CONCAT("group_key_", CAST(RAND()*10 AS INT64)) as group_key,
+    1000000*x2 + x1 as x
+from UNNEST(GENERATE_ARRAY(1,1000000)) as x1,
+     UNNEST(GENERATE_ARRAY(4,6)) as x2;
+```
+2.Groups on a certain key and aggregates theta_sketches per key
+```sql
+CREATE OR REPLACE TABLE `$BQ_PROJECT.$BQ_DATASET`.theta_agg_by_key_set_a AS
+select
+    group_key,
+    count(*) as total_count,
+    `$BQ_PROJECT.$BQ_DATASET`.theta_sketch_int64(x,14) as theta_sketch
```

Comment:
Now that we're publishing all the UDFs and UDAFs in the bqutil.fn dataset, let's save the user some manual work and just have all the examples in this README reference the public UDFs

Reply:
Updated